apps/codeforafrica/src/lib/data/common/index.js

```diff
@@ -44,11 +44,113 @@ function getPageSlug({ params }) {
   return params?.slugs?.[pageSlugIndex] || "index";
 }
 
+function getDefaultErrorPageProps(statusCode = 404) {
+  if (statusCode === 500) {
+    return {
```

Comment:
```suggestion
function getDefaultErrorPageProps(slug = "404) {
  if (slug === "500") {
    return {
```

Why `slug` to `statusCode` if `slug` can do?